src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Unix.cs

```diff
@@ -530,5 +534,157 @@ public static string[] GetLogicalDrives()
         {
             return DriveInfoInternal.GetLogicalDrives();
         }
+
+        /// <summary>Gets the path of the target of the specified link.</summary>
+        /// <param name="linkPath">A path to a link file.</param>
+        /// <returns>If linkPath represents a link file and it exists, returns the link's target path.
+        /// If linkPath is not a link or the target does not exist, returns null.</returns>
+        internal static string? GetLinkTarget(string linkPath) =>
+            LinkExists(linkPath, out _) ? Interop.Sys.ReadLink(linkPath) : null;
+
+        internal static void CreateSymbolicLink(string linkPath, string target, bool isDirectory)
+        {
+            if (linkPath == null)
+            {
+                throw new ArgumentNullException(nameof(linkPath));
+            }
+            else if (linkPath.Length == 0)
+            {
+                throw new ArgumentException(SR.Arg_PathEmpty_Name, nameof(linkPath));
```

Comment:
```suggestion
                throw new ArgumentException(SR.Arg_PathEmpty, nameof(linkPath));
```
Otherwise, the exception message will say "The path '{0}' is empty."